examples/step-60/step-60.cc

```diff
@@ -286,11 +309,12 @@ namespace Step60
     // Embedded dof handler
     std::unique_ptr<DoFHandler<dim, spacedim> > embedded_configuration_dh;
 
+    // Embedded mapping
+    std::unique_ptr<Mapping<dim,spacedim> > embedded_mapping;
+
     // The configuration vector
     Vector<double> embedded_configuration;
```

Comment:
embedded_mapping depends on embedded_configuration, therefore it should morally be defined afterwards. Revert the order change.